.github/workflows/github-actions-test.yml

```diff
@@ -0,0 +1,28 @@
+name: test
+on : [pull_request, push]
+jobs:
+  Explore-Github-Actions:
+    runs-on: macos-latest
+    steps:
+      - run: echo "The name of your branch is ${{ github.ref }} and your repository is ${{ github.repository }}."
+      - name: Checkout repository code
+        uses: actions/checkout@v2
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with: 
+          python-version: '3.7.7'
```

Comment:
Is there any reason to use a specific patch version?  Why not just '3.7'?

Reply:
No reason to use a specific version(Used it as it is verified on my local env)!! changed it to '3.7' and it works well.